packages/server/src/index.ts

```diff
@@ -1440,7 +1440,7 @@ export class App {
                 chatType,
                 chatId,
                 memoryType: memoryType ?? (chatId ? IsNull() : undefined),
-                sessionId: sessionId ?? (chatId ? IsNull() : undefined),
+                sessionId: sessionId ?? undefined,
```

Comment:
I think we can just do `sessionId,` since `sessionId: sessionId ?? undefined,` are the same as `sessionId,` right?

Reply:
I think if it is null, the sessionId will still be undefined.

Reply:
> I think if it is null, the sessionId will still be undefined.

what do you mean? if `sessionId` is null, it will still be undefined?

Reply:
Yes. The previous logic was that if the chatId was passed without the sessionId, the sessionId would be set to null. This would result in querying messages that have the same chatId but a null sessionId. @HenryHengZJ